content/captions.fr.md

```diff
@@ -0,0 +1,256 @@
+---
+title: "Légendes/Sous-titres"
+title_image: /content-images/wai-media-guide/cc.svg
+nav_title: "Légendes/Sous-titres"
+
+lang: fr   # change "en" to lang code, here and 2 @@s below
+last_updated: 2020-01-01   # Change to date of translation YYYY-MM-DD (month in middle)
+translators:
+- name: "Sofia Ahmed"
+contributors:
+- name: "Sandra Velarde Gonzalez (ETNIC)"
+
+permalink: /media/av/captions/fr  # Add lang to end /link/to/page/@@
+ref: /media/av/captions/   # Do not change this
+layout: default
+github:
+   repository: w3c/wai-media-guide
+   path: 'content/captions.fr.md'   # Add lang to the middle of the filename, e.g., index.@@.md
+
+resource:
+  ref: /media/av/
+navigation:
+  previous: /media/av/description/
+  next:     /media/av/transcripts/
+changelog: /media/av/changelog/
+acknowledgements: /media/av/acknowledgements/
+
+description: Vous aide à comprendre et à créer des légendes (aussi appelées "sous-titres") pour l'accessibilité des contenus audios et vidéos.
+image: /content-images/wai-media-guide/social.png
+
+footer: >   # Translate words below, including "Date:" and "Editor:". (Do not update the date.)
+   <p><strong>Date :</strong> Mise à jour : 12 septembre 2020. JOURNAL DES MODIFICATIONS.</p>
+   <p><strong>Rédactrice :</strong> <a href="http://www.w3.org/People/Shawn">Shawn Lawton Henry</a>. REMERCIEMENTS : liste les contributeurs et les crédits.</p>
```

Comment:
We should not translate "ACKNOWLEDGEMENTS" as it is automatically translated in the process.